go/chat/crypto_doc.md

```diff
@@ -0,0 +1,205 @@
+# Chat Crypto Doc
+
+## High Level Overview
+
+When Alice sends a message to Bob, she uses the same keys that she would use to
+save a file in `/keybase/private/alice,bob`. That makes a lot of nice things
+work right off the bat:
+
+- Alice can send a message to Bob even if he hasn't joined Keybase yet. If she
+  knows Bob is @bobbymcferrin on Twitter, she can send messages to him now, and
+  later one of her devices will check @bobbymcferrin's proof and send him keys
+  automatically.
+- If either of them adds a new device, that device will get a copy of all the
+  keys it needs to read old messages.
+- If either of them removes a device, all of their other devices will start
+  using a new encryption key. That guarantees the removed device can't read new
+  messages, without relying on the server to enforce it.
+
```

Comment:
You should say here what the high level encryption strategy is, like in 1-2 sentences: "A shared symmetric key for all participants, encrypted to each participant's public key.  The clients regenerate the key whenever a participant revokes a device."

Otherwise it just jumps right in at the middle.